src/component/user/Login/Login.tsx

```diff
@@ -15,9 +15,10 @@ const Login = () => {
     const { user } = useAuthUser();
     const query = useQueryParams();
     const resetPassword = query.get('reset') === 'true';
```

Comment:
Where does this get set, and will it overwrite or add to the query params? Worst case we will redirect to "/", but would be nice to check what happens.